scipy/optimize/_linprog_util.py

```diff
@@ -1046,13 +1069,8 @@ def vstack(blocks):
         zeros = np.zeros
         eye = np.eye
 
-    # bounds will be modified, create a copy
+    # Rows will be reversed, which feeds back into bounds, so copy
```

Comment:
Could you explain this?

Reply:
I'd better write a clearer comment :-)
What I mean to say is that `lbs` and `ubs`, introduced 2 lines down, may change, and these changes feed back into `bounds`.